src/core/libmaven/jsonReports.cpp

```diff
@@ -67,8 +67,7 @@ void JSONReports::_writeCompoundLink(PeakGroup& grp, ofstream& filename)
     filename << "\"compoundId\": "<< _sanitizeJSONstring(compoundID) ;
     string compoundName = grp.getCompound()->name;
     filename << ",\n" << "\"compoundName\": "<< _sanitizeJSONstring(compoundName);
-    string formula = grp.getCompound()->formula;
-    filename << ",\n" << "\"formula\": "<< _sanitizeJSONstring(formula);
+    string formula = grp.getCompound()->formula();
    filename << ",\n" << "\"formula\": "<< _sanitizeJSONstring(formula);

```

Comment:
Please check

Reply:
This is a harmless (as far as compilation is concerned) alternative to `\n`, and comes from Windows systems, they use carriage-return character for newlines. I will still get rid of it though. Thanks for pointing out.